dev/io.openliberty.io.opentelemetry.internal/bnd.bnd

```diff
@@ -28,26 +28,37 @@ Import-Package: \
   zipkin2.internal,\
   zipkin2.reporter,\
   zipkin2.reporter.okhttp3,\
-  io.opentelemetry.api.trace,\
-  io.opentelemetry.api,\
-  io.opentelemetry.api.baggage,\
-  io.opentelemetry.context
+  io.opentelemetry.api; version=${openTelemetryVersion},\
+  io.opentelemetry.api.baggage; version=${openTelemetryVersion},\
+  io.opentelemetry.api.common; version=${openTelemetryVersion},\
+  io.opentelemetry.api.trace; version=${openTelemetryVersion},\
+  io.opentelemetry.context; version=${openTelemetryVersion},\
+  io.opentelemetry.sdk.trace.export;version=${openTelemetryVersion},\
+  io.opentelemetry.sdk.trace.data;version=${openTelemetryVersion},\
+  io.opentelemetry.sdk.common;version=${openTelemetryVersion},\
+  io.opentelemetry.sdk.autoconfigure.spi.traces;version=${openTelemetryVersion},\
+  io.opentelemetry.sdk.autoconfigure.spi;version=${openTelemetryVersion},\
+  io.opentelemetry.sdk.resources;version=${openTelemetryVersion},\
+  io.opentelemetry.semconv.trace.attributes;version=${openTelemetryVersion},\
+  io.opentelemetry.semconv.resource.attributes;version=${openTelemetryVersion},\
+  io.opentelemetry.instrumentation.annotations;version=${openTelemetryVersion}
```

Comment:
The versions here are not quite right. We could manually set each one to `version="[${openTelemetryVersion},2)", but if we remove the version from here and add the API bundle to the buildpath then BND should create the correct version range automatically.

Reply:
right, not all of these are going to be version 1.19.0 ... for example some may well be 1.19.0-alpha

Reply:
Would it be best to remove the versions from this list then?